lib/util.js

```diff
@@ -267,7 +267,7 @@ exports.parseParam = function (val, spec) {
       val = exports.parseParams(val, spec.fields || {})
       break
     case 'boolean':
-      if (val === 'true' || val === 'false') return val === 'true'
+      if (val === 'true' || val === 'false') val === 'true'
```

Comment:
Why drop the `return`?

Reply:
i wasnt sure exactly where the best place for the return statement was, and val returns at the end of the function.

Reply:
But as written, I don't think `val` is set to anything?